src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
+    let start = 0;
+
+    if (arguments.length < 2) {
```

Comment:
```suggestion
    if (initialValue === undefined) {
```

Reply:
Can't use this recommendation because tests require code to handle undefined if it was passed as initialValue. Here you can see this test: https://prnt.sc/PgHvDcV4Fqgb